moderation/EmoteFilter.go.tmpl

```diff
@@ -0,0 +1,123 @@
+{{/*
+	Trigger ^.{0,2}(((<a?:[\w~]{2,32}:\d{17,19}>)|[\x{1f1e6}-\x{1f1ff}]{2}|\p{So}\x{fe0f}?[\x{1f3fb}-\x{1f3ff}]?(\x{200D}\p{So}\x{fe0f}?[\x{1f3fb}-\x{1f3ff}]?)*|[#\d*]\x{FE0F}?\x{20E3}).{0,2}|\s+)+$
```

Comment:
Note that `^` and `$` match the start and end of line, not start and end of string. Not sure if intended. If not, use `\A` and `\z` instead.

Reply:
This actually was intentional, I wanted to catch those annoying multiline emote shapes that people do, using whole string makes it easier to bypass by adding a few periods or something onto the end. At least with by-line they would need to add characters to every line. Nothing is going to be fool proof but if you make it annoying enough to bypass it can be effective

ex: this would bypass using `\A` `\z` whereas `^` `$` will still catch it
😂 
😂😂
😂😂😂
😂😂😂😂
😂😂😂
😂😂
😂.....

Reply:
Hm. I feel modifying the regex/doing some checks in the code itself with a more permissive regex might be useful to consider in a followup PR. Not a blocker though, so gonna go ahead and merge this 😄